spec/features/courses/age_range_in_years/new_spec.rb

```diff
@@ -25,21 +29,36 @@
                        level: :primary)
     stub_api_v2_new_resource(new_course)
     stub_api_v2_resource(recruitment_cycle)
-    stub_api_v2_resource_collection([new_course], include: "sites,provider.sites,accrediting_provider")
+    stub_api_v2_resource_collection([new_course], include: "subjects,sites,provider.sites,accrediting_provider")
     stub_api_v2_build_course
+    stub_api_v2_build_course(age_range_in_years: "3_to_7")
   end
 
   scenario "sends user to entry requirements" do
-    visit new_provider_recruitment_cycle_courses_age_range_path(
-      provider.provider_code,
-      provider.recruitment_cycle_year,
-    )
+    visit_new_age_range_page
 
-    stub_api_v2_build_course(age_range_in_years: "3_to_7")
     choose("course_age_range_in_years_3_to_7")
-    click_on "Continue"
-
+    new_age_range_page.continue.click
 
     expect(new_outcome_page).to be_displayed
   end
+
+  scenario "sends user back to course confirmation" do
+    visit_new_age_range_page(goto_confirmation: true)
+
+    choose("course_age_range_in_years_3_to_7")
+    new_age_range_page.continue.click
+
+    expect(current_path).to eq confirmation_provider_recruitment_cycle_courses_path(provider.provider_code, provider.recruitment_cycle_year)
+  end
+
+  def visit_new_age_range_page(**query_params)
+    2.times do
```

Comment:
As mentioned, could we visit the sign-in page before we hit this one to make it clear what we're doing?

Reply:
oh right so user not quite authenticated and getting sent in circles